docs/09_Pages/Navigating_Pages/Internal_Page_on_App_Load.md

```diff
@@ -0,0 +1,37 @@
+The default behavior for DrupalGap is to direct to the front page set in the `settings.js` file. This breaks the sharing feature that users are used to on a typical web site. This code may be added to `settings.js` to enable direction to internal pages.
+
+**This method has only been tested in a web app.**
+
+```
+var landingPage = 'dashboard';
+if(location.hash){
+  // additional logic can be added here for other internal page types and exclude certain pages from being directed to on the initial app load.
+  goToPage = location.hash.replace("#","");
+  landingPage = goToPage.replace('node_', 'node/');
+}
+
+// fix drupalgap_back() to not break if coming from external page
```

Comment:
There was some concern with [pull request 787](https://github.com/signalpoint/DrupalGap/pull/787) that this change might break the back button for an app ported through PhoneGap. I'm thinking that this solution only works for web apps, in a native app you'll have to parse data from a notification to set a dynamic front page. Notice that I hardcoded the page on line 28 of this code. If it's added to the core, it will probably need to have another variable set there, something like this maybe?

Starting on line 27 of this code: 
```
else if (drupalgap.back_path == ''){
    drupalgap_goto('drupalgap.settings.default_back');
    return;
}
```

And require that `drupalgap.settings.default_back` is set in `settings.js`